stm32-lvgl/Sources/Support/Stubs.c

```diff
@@ -0,0 +1,24 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors.
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+#include <string.h>
+#include <stdint.h>
+
+int getentropy(char *buffer, size_t length) {
```

Comment:
lets add a disclaimer to this implementation?

Reply:
added